backend/corpora/lambdas/api/v1/curation/collections/common.py

```diff
@@ -96,12 +96,11 @@ def reshape_datasets_for_curation_api(datasets: typing.List[dict], preview=False
 
 
 def reshape_dataset_for_curation_api(dataset: dict, preview=False) -> dict:
-    if artifacts := dataset.get("artifacts"):
+    if artifacts := dataset.pop("artifacts", []):
```

Comment:
nit: since this is an operation with a side effect, I think It would be better to split this into two separate statements: the pop and the if. Just an opinion though.

Reply:
with the walrus operator (`:=`) we are doing both of those operations in one.